02-electric/electric-section-three/electric-section-three.html

```diff
@@ -9,37 +9,97 @@
   <title>01 - Electric - Section Three</title>
 </head>
 <body>
-
   <section class="electric-section-three-container">
+
     <section class="electric-section-three-subcontainer">
-      <div class="electric-section-three-img-container">
-        <img
-         class="electric-section-three-img"
-         src="../../00-overall/05-images/placeholder.svg"
-         alt="image section four"
-        />
-      </div>
+
       <h1 class="electric-section-three-title">
-      This is title of this section
+        Don’t forget these energy saving tips
       </h1>
-      <p class="electric-section-three-content">
-      it is a long established fact that a reader will be distracted by the
-      readable content of a page when looking at its layout. The point of
-      using Lorem Ipsum is that it has a more-or-less normal distribution of
-      letters, as opposed to using 'Content here, content here', making it
-      look like readable English. Many desktop publishing packages and web
-      page editors now use Lorem Ipsum as their default model text, and a
-      search for 'lorem ipsum' will uncover many web sites still in their
-      infancy. Various versions have evolved over the years, sometimes by
-      accident, sometimes on purpose (injected humour and the like).
-      </p>
-      <div class="electric-section-three-small-content-container" >
-      <img class="electric-section-three-small-img" src="../../00-overall/05-images/placeholder.svg" alt="small image section four">
-      <p class="electric-section-three-small-prg">Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+
+      <div class="electric-section-three-part-one-container">
```

Comment:
can we rename every "part" to "subsection" please:

for example:

"electric-section-three-subsection-one-container"
"electric-section-three"subsection-two-container"
"electric-section-three"subsection-three-container"

you can use Find + Replace in VSCode
press Ctrl + H
in the top bar is the original text
and the bottom bar is the replacement text

so for example: 
top bar: -part-one-
bottom bar: -subsection-one-

then press ctrl + alt + enter to Replace All

(you will have to do the same in css)

Reply:
done